resource/constants.rs

```diff
@@ -61,7 +61,7 @@ pub mod server {
 pub mod database {
     use std::time::Duration;
 
-    pub const QUERY_PLAN_CACHE_FLUSH_STATISTICS_CHANGE_PERCENT: f64 = 0.05;
+    pub const QUERY_PLAN_CACHE_FLUSH_ANY_STATISTIC_CHANGE_FRACTION: f64 = 0.25;
```

Comment:
Raised this to 25% since it now applies to _any_ individual entity/relation/attribute/has/relation->role/role->player statistic! 

In general what we want to ensure is that if we have a big increase in "cost" of planning, we replan. Critically, this means we really need to replan when we go from say 1 -> 2 elements (since a plan which includes 5 stacked 1 element iteration steps is going to behave quite different to 5 stacked 2 element iteration steps = 2x2x2x2x2 = 32 elements! The bigger the diff between the old plan and the new number of elements, the worse can plan can be, obviously!)